crates/driver/src/logic/competition/order/signature.rs

```diff
@@ -0,0 +1,17 @@
+/// Signature over the order data.
+#[derive(Debug)]
+pub struct Signature {
```

Comment:
For the domain object, it might make sense for the `from` address to also be part of the signature (since, AFAIK, it is only ever used in the context of encoding a signature by the driver).

Reply:
My point was that `from` isn't really order data, but data inherent to the signature:
- for `eip712` and `ethsign`, it is the recovered address
- for `eip1271` and `presign`, it needs to be included in the signature bytes 

The `from` is actually not part of order data from the smart contract point of view.

I think of the `from` more as a `signer` anyway - which would fit a little better with the `Signature` domain object.

This wasn't a strong suggestion, just thrown out there since I know that when encoding the signature for settlement calldata, the `signer` address is needed. I think it is fine as is though.